deployer/src/main/java/eu/seaclouds/policy/SeaCloudsInitializerPolicy.java

```diff
@@ -0,0 +1,544 @@
+
+    private String observerInitializerPayload;
+    private String grafanaInitializerPayload;
+
+    //TODO: This method is boilerplate code also contained in eu.seaclouds.platform.dashboard.util.ObjectMapperHelpers
+    private <T> T xmlToObject(String xml, Class<T> type) throws JAXBException {
+        JAXBContext jaxbContext = JAXBContext.newInstance(type);
+        Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
+
+        T obj = (T) jaxbUnmarshaller.unmarshal(new StringReader(xml));
+        return obj;
+    }
+
+    // Helper method to extract metric names from Monitoring Rules
+    private List<String> extractMetricNames() {
+
+        List<String> result = new ArrayList<>();
+        for (MonitoringRule rule : monitoringRules.getMonitoringRules()) {
+
+            /*
+             * Workaround loop to retrieve the metric name. This is required due a poorly modeled POJO
+             * <actions>
+             *     ...
+             *     <action name="OutputMetric">
+             *         <parameter name="metric">appid_mysql_server_cpu_utilization_metric</parameter>
+             *          ...
+             *      </action>
+             * </actions>
+             */
+
+            for (Action action : rule.getActions().getActions()) {
+                if (action.getName().equalsIgnoreCase("OutputMetric")) {
+                    for (Parameter parameter : action.getParameters()) {
+                        if (parameter.getName().equalsIgnoreCase("metric")) {
+                            result.add(parameter.getValue());
+                        }
+                    }
+                }
+            }
+        }
+        return result;
+    }
+
+    private void parseMonitoringRules() {
+        String decodedRules = new String(BaseEncoding.base64().decode(getConfig(T4C_RULES)));
+        try {
+            monitoringRules = xmlToObject(decodedRules, MonitoringRules.class);
+        } catch (JAXBException e) {
+            Exceptions.propagateIfFatal(e);
+            LOG.warn("Cannot unmarshal the content of t4c.rule into MonitoringRules");
+        }
+    }
+
+    private void parseAgreements() {
+        String decodedRules = new String(BaseEncoding.base64().decode(getConfig(SLA_AGREEMENT)));
+        try {
+            agreement = xmlToObject(decodedRules, Agreement.class);
+        } catch (JAXBException e) {
+            Exceptions.propagateIfFatal(e);
+            LOG.warn("Cannot unmarshal the content of sla.agreement into an Agreement");
+        }
+    }
+
+    private void setupObserverInitializerPayload() {
+        // T4C JSON payload to attach a INFLUXDB Observer to T4C
+        observerInitializerPayload = "{ \"format\":\"INFLUXDB\", \"protocol\":\"HTTP\",\"callbackUrl\":" +
+                "\"" + getConfig(INFLUXDB_ENDPOINT) + "/db/" + getConfig(INFLUXDB_DATABASE) +
+                "/series?u=" + getConfig(INFLUXDB_USERNAME) + "&p=" + getConfig(INFLUXDB_PASSWORD) + "\" }";
+    }
+
```

Comment:
I'm already aware about Grafana Scripted Dashboads, but this is a quick solution which doesn't involve custom JS at Grafana side.